app/src/main/java/com/sopt/now/UserInfo.kt

```diff
@@ -0,0 +1,10 @@
+package com.sopt.now
+
+import java.io.Serializable
+
+data class UserInfo(
+    var id : String = "",
+    var password : String = "",
+    var nickname : String = "",
+    var mbti : String = ""
+) : Serializable

```

Comment:
```suggestion
data class UserInfo(
    var id : String,
    var password : String,
    var nickname : String,
    var mbti : String
) : Serializable

```
초기값을 설정해 줄 필요가 없다면 지정해주지 않아도 됩니다 !